README.md

```diff
@@ -46,6 +46,7 @@ Here's how the default settings look like:
     static_url_base: 'https://ucarecdn.com',
     api_version: '0.5',
     cache_files: true,
+    store_files_upon_uploading: false,
```

Comment:
@dmitry-mukhin the only place where `autostore` word is mentioned is docs' "[Widget](https://uploadcare.com/documentation/widget/)" section. I haven't saw it since I've never read this section. The correspondent setting in a project dashboard is called "Automatic file storing". Also you have `UPLOADCARE_DO_NOT_STORE` widget flag, `UPLOADCARE_STORE` flag for file uploads API and `store` flag for url uploads API. 

I perfectly agree with your comment, there is no need to add another name)